easybuild/easyconfigs/p/PyTorch/PyTorch-1.10.1-CrayGNU-21.09.eb

```
@@ -0,0 +1,85 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'PythonBundle'
+
+name = 'PyTorch'
+version = '1.10.1'
+_ptcudaver = 'cu113'
```

This is probably fine, but should we use 11.1 and cudatoolkit/11.1.0_3.39-4.1__g484e319 for consistency?
```suggestion
_ptcudaver = 'cu111'
```
Available wheels: https://download.pytorch.org/whl/torch_stable.html

---

I wished to try the latest `CUDA 11.2` available on Dom and it makes the test pass with the extensions `cu113` provided by `torch` (tested locally with the modulefile in my sandbox).

---

hm... it might be fine, but if we can, it might be better to use the same cuda version everywhere. Here you are building Horovod with cuda 11.2 but using torch built with 11.3, which isn't even available for us yet. I guess that could cause some problems for users trying to build [torch extensions](https://pytorch.org/tutorials/advanced/cpp_extension.html) or some libraries like Nvidia APEX, MS DeepSpeed, etc

---

I have switched the `torch` extensions to `cu111` as you suggested @henrique: I load the default `cudatoolkit` now, that is 11.0. The regressions tests pass also in this case, it seems that minor CUDA versions do not make any difference there.